dev/NavigationView/NavigationView.cpp

```diff
@@ -2422,6 +2588,11 @@ void NavigationView::OnKeyDown(winrt::KeyRoutedEventArgs const& e)
     case winrt::VirtualKey::GamepadRightShoulder:
         handled = BumperNavigation(1);
         break;
+    case winrt::VirtualKey::Tab:
+        // arrow keys navigation through ItemsRepeater don't get here
+        // so handle tab key to distinguish between tab focus and arrow focus navigation
+        m_TabKeyPrecedesFocusChange = true;
```

Comment:
Does this behavior work if you hold down the tab key?

Reply:
It does.